website/project/model.py

```diff
@@ -179,6 +179,33 @@ class MetaData(GuidStoredObject):
     date_modified = fields.DateTimeField(auto_now=datetime.datetime.utcnow)
 
 
+def validate_contributor(guid, contributors):
+    if guid:
```

Comment:
The way the regex in `get_valid_mentioned_users_guids` is written, I don't believe it's possible to have `''` for a guid here. If it *is* possible, then the way this is written will mean that any empty strings will be included in the return value of `get_valid_mentioned_users_guids`, which seems non-ideal. A couple options:

 - Remove this condition, and assume that `guid` will never be empty (though if it is, what will the effect be?)
  - We could explicitly filter out empty guids in the `new_mentions` list comprehension assignment on line 202-206.
 - Raise `ValidationValueError` when guid is empty.